web/src/main/webapp/WEB-INF/jsp/managestudy/viewSectionDataPreview.jsp

```diff
@@ -15,318 +15,317 @@
+            <img src="images/icon_InitialDE.gif" alt="<fmt:message key="initial_data_entry" bundle="${resword}"/>" title="<fmt:message key="initial_data_entry" bundle="${resword}"/>">
+        </c:when>
+        <c:when test="${EventCRFBean.stage.name=='initial data entry complete'}">
+            <img src="images/icon_InitialDEcomplete.gif" alt="<fmt:message key="initial_data_entry_complete" bundle="${resword}"/>" title="<fmt:message key="initial_data_entry_complete" bundle="${resword}"/>">
+        </c:when>
+        <c:when test="${EventCRFBean.stage.name=='double data entry'}">
+            <img src="images/icon_DDE.gif" alt="<fmt:message key="double_data_entry" bundle="${resword}"/>" title="<fmt:message key="double_data_entry" bundle="${resword}"/>">
+        </c:when>
+        <c:when test="${EventCRFBean.stage.name=='data entry complete'}">
+            <img src="images/icon_DEcomplete.gif" alt="<fmt:message key="data_entry_complete" bundle="${resword}"/>" title="<fmt:message key="data_entry_complete" bundle="${resword}"/>">
+        </c:when>
+        <c:when test="${EventCRFBean.stage.name=='administrative editing'}">
+            <img src="images/icon_AdminEdit.gif" alt="<fmt:message key="administrative_editing" bundle="${resword}"/>" title="<fmt:message key="administrative_editing" bundle="${resword}"/>">
+        </c:when>
+        <c:when test="${EventCRFBean.stage.name=='locked'}">
+            <img src="images/icon_Locked.gif" alt="<fmt:message key="locked" bundle="${resword}"/>" title="<fmt:message key="locked" bundle="${resword}"/>">
+        </c:when>
+        <c:otherwise>
+            <img src="images/icon_Invalid.gif" alt="<fmt:message key="invalid" bundle="${resword}"/>" title="<fmt:message key="invalid" bundle="${resword}"/>">
+        </c:otherwise>
+    </c:choose>
+</c:if>
   </span></h1></span>
 
 <script type="text/javascript" language="JavaScript">
-// <![CDATA[
-function getSib(theSibling){
-  var sib;
-  do {
-   sib  = theSibling.previousSibling;
-   if(sib.nodeType != 1){
-     theSibling = sib;
-   }
-  } while(! (sib.nodeType == 1))
-
-  return sib;
-}
-// ]]>
+    // <![CDATA[
+    function getSib(theSibling) {
+        var sib;
+        do {
+            sib = theSibling.previousSibling;
+            if (sib.nodeType != 1) {
+                theSibling = sib;
+            }
+        } while (!(sib.nodeType == 1))
+
+        return sib;
+    }
+    // ]]>
 </script>
 
 
 <c:if test="${studySubject != null && studySubject.id>0}">
-<p>
-<div class="homebox_bullets"><a href="ViewEventCRF?id=<c:out value="${EventCRFBean.id}"/>&studySubId=<c:out value="${studySubject.id}"/>">View Event CRF Properties</a></div>
-<p>
-<p>
-<div class="homebox_bullets" style="width:117">
+    <p>
+    <div class="homebox_bullets"><a href="ViewEventCRF?id=<c:out value="${EventCRFBean.id}"/>&studySubId=<c:out value="${studySubject.id}"/>">View Event CRF Properties</a></div>
+    <p>

```

Comment:
This is a hardcoded English string that should be updated to use the properties file.